edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/KubernetesDeploymentMapperTest.cs

```diff
@@ -745,6 +784,45 @@ public void PassImagePullSecretsInPodSpecForProxyAndModuleContainers()
             Assert.Contains(deployment.Spec.Template.Spec.ImagePullSecrets, secret => secret.Name == "user-registry2");
         }
 
+        [Fact]
+        public void FailMapWhenMountWhenMountTypeIsMissing()
+        {
+            var identity = new ModuleIdentity("hostname", "gatewayhost", "deviceid", "Module1", Mock.Of<ICredentials>());
+            var hostConfig = NoTypeVolumeMountHostConfig;
```

Comment:
Do we have a test case in which `hostConfig = null`?

Reply:
The `NoMountsNoClaims` test uses default (=null) `HostConfig`.  The `CreatePodParameters` class keeps `HostConfig` as an `Option`.